### PR TITLE
Disable add_output_performance test

### PR DESCRIPTION
**Pull request by olpipi**
### Details:
 - Disable add_output_performance test

### Tickets:
 - CVS-140440
 - CVS-140158
 - CVS-134286

### CODE REVIEW

**Review by p-durandin — APPROVED** [review]

File: src/core/tests/model.cpp [hunk 1]
```diff
@@ -1147,6 +1147,8 @@ TEST(model, add_output_port_to_result) {
 }
 
 TEST(model, add_output_performance) {
+    // skip this test due to CVS-140440
+    GTEST_SKIP();
```
Comment (iefode) [inline src/core/tests/model.cpp:1151]:
```suggestion
    SKIP_IF_CURRENT_TEST_IS_DISABLED();
```

**Review by iefode — COMMENTED** [review]
Add test to skipped test list

**Comment by olpipi** [comment]
> Add test to skipped test list

There is no such list in ov_core_unit_tests. That's why I added skip to test body

File: src/core/tests/model.cpp (hunk 1 above)
Comment (olpipi) [reply src/core/tests/model.cpp:1151]: SKIP_IF_CURRENT_TEST_IS_DISABLED is defined in func test utils.
I don't want to add dependency to func test utils to unit test target.

Comment (vurusovs) [reply src/core/tests/model.cpp:1151]:
```suggestion
    GTEST_SKIP() << "CVS-140440";
```

**Review by vurusovs — APPROVED** [review]

**Pull request merged**